packages/esbuild/esbuild.bzl

```diff
@@ -115,15 +115,17 @@ def _esbuild_impl(ctx):
     if ctx.attr.max_threads > 0:
         env["GOMAXPROCS"] = str(ctx.attr.max_threads)
 
+    execution_requirements = {}
+    if "no-remote-exec" in ctx.attr.tags:
```

Comment:
is this a breaking change, in that users will have to add this tag after the next release?

@mattem I'm not clear why no-remote-exec was required here, do you know?

Reply:
Only if those users have RBE enabled, and don't want this particular action to happen on RBE for whatever reason. While the behavior may change (for those users), it shouldn't really break anyone's build.

Discussed this a little on Friday with @mattem on Slack, he wasn't sure why that execution_requirements was set initially:
https://bazelbuild.slack.com/archives/CEZUUKQ6P/p1616796872102700

I did test this change on RBE and the rule works just fine remotely.

Happy to do this in a different way if you prefer and make disabling RBE the default - just seems like the tag approach is somewhat standard.

Reply:
I had a think about this, and I think it's just some copypasta from when I copied it from evertz internal.
If this is working as expected on RBE then it should be fine, but there is the point about breaking change, albeit small imo.

Reply:
Let me know what you think the best way to proceed is - happy to make any changes or mark the PR as introducing a breaking change if you prefer.

Would love to get this landed, but can always stick with the local fork approach until we're ready!

Reply:
If we were really worried about the breaking change, we'd make your new attribute default match the current behavior. But I don't think we are, plus the esbuild package is quite new so I'm less concerned about dormant code that depends on it.